runner/buckets.go

```diff
@@ -1,6 +1,15 @@
 package runner
 
 var customBuckets = map[string][]float64{
+	"gateway.request_size": {
+		10,       // 10 bytes
+		100,      // 100 bytes
+		1000,     // 1kb
+		10000,    // 10kb
+		100000,   // 100kb
+		1000000,  // 1mb
+		10000000, // 10mb
+	},
```

Comment:
[optional]: Maybe with `bytesize` we won't need the comments?
```suggestion
	"gateway.request_size": {
		float64(10 * bytesize.B),
		float64(100 * bytesize.B),
		float64(1 * bytesize.KB),
		float64(10 * bytesize.KB),
		float64(100 * bytesize.KB),
		float64(1 * bytesize.MB),
		float64(10 * bytesize.MB),
	},
```

Reply:
Good idea :+1: